Assignment _5/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.3)
+project(Assignment__5)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+
+set(SOURCE_FILES main.cpp main.h reader.cpp)
+add_executable(Assignment__5 ${SOURCE_FILES})
```

Comment:
Don't forget to link your library if this is you "driver"
`target_link_libraries(Assignment__5 <your namespace>)`